eng/Versions.props

```diff
@@ -45,6 +45,9 @@
     <CLI_NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-26011-01</CLI_NETStandardLibraryNETFrameworkVersion>
     <SharedHostVersion>$(MicrosoftNETCoreAppPackageVersion)</SharedHostVersion>
     <HostFxrVersion>$(MicrosoftNETCoreAppPackageVersion)</HostFxrVersion>
+    <!-- <TargetingPackVersion>$(MicrosoftNETCoreAppPackageVersion)</TargetingPackVersion> -->
+    <TargetingPackVersion>3.0.0-preview4-27419-5</TargetingPackVersion>
```

Comment:
Rename this to NetCoreAppTargetingPackVersion (or similar)

Reply:
Completed...